infrastructure/module/eks/sa.helm_release.tf

```diff
@@ -32,7 +32,7 @@ resource "helm_release" "alb-controller" {
   }
 
   set {
-    name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
```

Comment:
You need role annotation but not webhookTLS.
Also check to add **defaultTargetType** to be **IP**.